media_driver/agnostic/common/os/mos_sw_swizzle.c

```diff
@@ -0,0 +1,1029 @@
+/*
+* Copyright (c) 2019, Intel Corporation
```

Comment:
If you copy from Mesa, you might check how you should credit them via license. Looks like you can copy the code, but copyright notice should be preserved. Here you miss "Copyright 2013 Google". You might also credit Intel with the correct years range since originally it is "Copyright 2012 Intel Corporation".

Reply:
ok